Deploy not working from Lektor Admin on Python 3.6

I'm trying to deploy Lektor from the admin interface running on Python 3.6. I get the following error in the terminal:

```
Started build
Error on request:
Traceback (most recent call last):
  File "/private/tmp/test/lib/python3.6/site-packages/werkzeug/serving.py", line 270, in run_wsgi
    execute(self.server.app)
  File "/private/tmp/test/lib/python3.6/site-packages/werkzeug/serving.py", line 261, in execute
    write(data)
  File "/private/tmp/test/lib/python3.6/site-packages/werkzeug/serving.py", line 241, in write
    assert isinstance(data, bytes), 'applications must write bytes'
AssertionError: applications must write bytes
Finished build in 0.02 sec
```

These are the steps to reproduce:

```
$ virtualenv -p python3.6 test
$ cd test/
$ source bin/activate
$ pip install lektor
$ lektor quickstart  # answer all with "test" and Yes
$ cd test

# add the following to test.lektorproject
[servers.ghpages]
name = GHPages
enabled = yes
target = ghpages://test/test

$ lektor server

# go to http://localhost:5000/admin/root/edit
# click Deploy icon
# observe error above in the terminal
```

I tried other deploy options as well and I get the same error. 

What else can I try? What additional information can I provide to make this easier to debug?

--- a/lektor/admin/utils.py
+++ b/lektor/admin/utils.py
@@ -16,7 +16,7 @@ def eventstream(f):
     def new_func(*args, **kwargs):
         def generate():
             for event in chain(f(*args, **kwargs), (None,)):
-                yield 'data: %s\n\n' % json.dumps(event)
+                yield ('data: %s\n\n' % json.dumps(event)).encode()
         return Response(generate(), mimetype='text/event-stream',
                         direct_passthrough=True)
     return update_wrapper(new_func, f)

--- a/tests/test_api.py
+++ b/tests/test_api.py
@@ -62,3 +62,16 @@ def test_children_sorting_via_api(scratch_project, scratch_env, children_records
     ordered_records = sorted(records_ordered_by_title, key=itemgetter('pub_date'), reverse=True)
 
     assert list(map(itemgetter('id'), ordered_records)) == children_records_ids_provided_by_api
+
+
+def test_eventstream_yield_bytes():
+    count = 0
+    from lektor.admin.utils import eventstream
+    @eventstream
+    def testfunc():
+        yield 'string'
+        yield 5
+    for data in testfunc().response: # pylint: disable=no-member
+        count += 1
+        assert isinstance(data, bytes)
+    assert count >= 2

